observability/WFLY-14947-Implement_observability_policy_metrics.adoc

```diff
@@ -0,0 +1,71 @@
+ = Implement Observability Policy for Metrics
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+While WildFly/EAP already supports basic metrics in the base WildFly/EAP system, as well as MicroProfile Metrics in the WildFly MicroProfile configuration and EAP XP, there is an effort in Red Hat Middleware Runtimes to implement a consistent observability policy across all Runtime offerings. In the Java space, the library of choice, and the emerging de facto standard for Java developers is https://micrometer.io[Micrometer].  This RFE will add support for Micrometer to WildFly/EAP and adhere to the policy decided upon by the various stakeholders.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-14947
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1686
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+* [X] Engineering
+* [X] QE
+
+=== Affected Projects or Components
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+* [x] Managed domain
```

Comment:
Please check but I don't think we support metrics on domain mode.

Reply:
> Please check but I don't think we support metrics on domain mode.

OK. That was probably a copy and paste from a template or prior document. Will verify if it's needed, but I think you're right.

Reply:
If wanted that should be a separate RFE. It brings with a big set of different considerations that would need to involve a lot of different people.